pkg/api/types.go

```diff
@@ -1345,6 +1345,16 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// KubeletPorts lists ports opened by Kubelet
+type KubeletPorts struct {
+	// KubeletPort on which Kubelet is listening
+	KubeletPort uint `json:kubeletPort`
+	// KubeletReadOnlyPorty only port open on Kubelet
+	KubeletReadOnlyPorty uint `json:kubeletReadOnlyPort`
```

Comment:
Porty is a typo. The idea behind this change is to make Node object everything's that needed to interact with 'things' (currently Kubelet/CAdvisor, I may add KubeProxy there as well) running on the Node. Adding RO and CAdvisor was pretty straightforward, so I did this. KubeProxy is a bit more tricky, so I left it for later. Those informations are generally for external consumption.